json-schema-core/src/main/java/io/ballerina/jsonschema/core/GeneratorUtils.java

```diff
@@ -123,12 +126,80 @@ public class GeneratorUtils {
 
     private static final String ITEM_SUFFIX = "Item";
     private static final String NAME_REST_ITEM = "RestItem";
+    private static final String REST_TYPE = "RestType";
+    private static final String ADDITIONAL_PROPS = "AdditionalProperties";
+    public static final String DEPENDENT_SCHEMA = "DependentSchema";
+    public static final String DEPENDENT_REQUIRED = "DependentRequired";
+    public static final String UNEVALUATED_PROPS = "UnevaluatedProperties";
+    public static final String PATTERN_ELEMENT = "PatternElement";
+    public static final String PATTERN_PROPERTIES = "PatternProperties";
+    public static final String UNEVALUATED_ITEMS_SUFFIX = "UnevaluatedItems";
+    public static final String PROPERTY_NAMES_SUFFIX = "PropertyNames";
 
     private static final ArrayList<String> STRING_FORMATS = new ArrayList<>(
             Arrays.asList("date", "time", "date-time", "duration", "regex", "email", "idn-email", "hostname",
                     "idn-hostname", "ipv4", "ipv6", "json-pointer", "relative-json-pointer", "uri",
                     "uri-reference", "uri-template", "iri", "iri-reference", "uuid")
     );
+    private static final ArrayList<String> BAL_PRIMITIVE_TYPES = new ArrayList<>(
+            Arrays.asList(INTEGER, STRING, BOOLEAN, NEVER, NULL, JSON)
```

Comment:
If the order matters in the implementation, this should be
integer, boolean, null, never, json, string